graphql/logging.ts

```diff
@@ -26,18 +22,6 @@ export const GraphQLLogger: any = {
         logger.info(`Started processing query`, { query });
 
         const handler: any = {
-            // Actually GraphQLRequestListener, but we're on v2 and not on v3
-            didEncounterErrors(requestContext: GraphQLRequestContext) {
-                const unexpected = requestContext.errors.some(isUnexpectedError);
```

Comment:
The errors in this case always have the same type `GraphQLError` and often miss the originalError field, which makes it hard to distinguish.

Reply:
I've checked the logs of the recent 15 days and there wasn't a single `Unexpected Errors`. In combination with type findings I would assume that the current logic doesn't work as expected.

We could log all the errors as info, so that we don't loose the information.